<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgInertiaTensor</class>
 <widget class="QDialog" name="CQDlgInertiaTensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inertia Matrix</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>310</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Inertia matrix divided by the mass</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="qqI33">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI23">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI21">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI32">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI22">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqI31">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>(The inertia matrix must be expressed relative
to the center of mass)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Position of the center of mass (COM)</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="qqMz">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqMx">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqMy">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>66</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="qqShapeFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inertia &amp;&amp; COM are relative to shape frame</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="qqWorldFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inertia &amp;&amp; COM are relative to absolute frame</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqI11</tabstop>
  <tabstop>qqI21</tabstop>
  <tabstop>qqI31</tabstop>
  <tabstop>qqI12</tabstop>
  <tabstop>qqI22</tabstop>
  <tabstop>qqI32</tabstop>
  <tabstop>qqI13</tabstop>
  <tabstop>qqI23</tabstop>
  <tabstop>qqI33</tabstop>
  <tabstop>qqMx</tabstop>
  <tabstop>qqMy</tabstop>
  <tabstop>qqMz</tabstop>
  <tabstop>qqShapeFrame</tabstop>
  <tabstop>qqWorldFrame</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
